src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -126,8 +126,8 @@ public void Execute_TargetElementLocatorReturnsNull_PassesNullToScan()
             _scanToolsMock.Setup(x => x.Actions).Returns(_actionsMock.Object);
             _scanToolsMock.Setup(x => x.NativeMethods).Returns(_nativeMethodsMock.Object);
 
-            _targetElementLocatorMock.Setup(x => x.LocateRootElements(It.IsAny<int>())).Returns<IEnumerable<A11yElement>>(null);
-            _actionsMock.Setup(x => x.Scan(null, It.IsNotNull<ScanActionCallback<ScanResults>>())).Returns<ScanResults>(null);
+            _targetElementLocatorMock.Setup(x => x.LocateRootElements(It.IsAny<int>())).Returns(CreateMockElementArray());
```

Comment:
This test is simulates the case where the `TargetElementLocator` returns `null`. That behavior shouldn't change

Reply:
Very good point, I think I fixed the test correctly now but of course you are more knowledgeable here.